src/ActiveLogin.Authentication.BankId.Api/Models/Request.cs

```diff
@@ -110,9 +110,10 @@ public Request(string endUserIp, string userVisibleData, byte[]? userNonVisibleD
     /// </param>
     /// <param name="returnUrl">The URL to return to when the authentication order is completed.</param>
     /// <param name="returnRisk">If set to true, a risk indication will be included in the collect response.</param>
-    public Request(string endUserIp, string? userVisibleData, byte[]? userNonVisibleData, Requirement? requirement, string? userVisibleDataFormat, string? returnUrl = null, bool? returnRisk = null)
+    /// <param name="deviceParameters">Information about the device the end user is using.</param>
+    public Request(string endUserIp, string? userVisibleData, byte[]? userNonVisibleData, Requirement? requirement, string? userVisibleDataFormat, string? returnUrl = null, bool? returnRisk = null, IBankIdEndUserDeviceData? deviceParameters = null)
```

Comment:
To harmonize with BankID API we decided to use the properties "web" and "app" instead of "deviceParameters". The params will be populated in the BankIdFlowService instead.